client/components/common/dashboardCard.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import RangeSlider from "./rangeSlider";
+import SwitchComponent from "./switch";
+class DashboardCardComponent extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
Move to a simple variable declaration instead of setting initial state in constructor